### PR TITLE
[German] Review of 2022 Wordle sentence_translations.json

### PR DESCRIPTION
**Pull request by Thadaeus**
Hello, I have revised the first 815 lines and changed a few things. If someone who is a native German speaker could check my changes and translations, I would be very grateful.

### CODE REVIEW

File: 2022/wordle/german/sentence_translations.json [hunk 1]
```diff
@@ -209,7 +209,7 @@
  {
   "input": "That's kind of a toss up between them at this point, so I guess probably just because it's alphabetical it goes with shard.",
   "model": "nmt",
-  "translatedText": "Das ist im Moment eine Art Streit zwischen ihnen, also denke ich, dass es wahrscheinlich nur, weil es alphabetisch ist, mit Shard zusammenhängt.",
+  "translatedText": "Das ist im Moment eine Art Unentschieden zwischen ihnen, also nehme ich an einfach weil es alphabetisch ist, dass es shard ist.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:212]:
```suggestion
  "translatedText": "Das ist im Moment eine Art Unentschieden zwischen ihnen, ich nehme also an, dass er shard nimmt, weil er alphabetisch vorgeht.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 2]
```diff
@@ -218,7 +218,7 @@
  {
   "input": "Which hooray, is the actual answer.",
   "model": "nmt",
-  "translatedText": "Hurra, ist die eigentliche Antwort.",
+  "translatedText": "welche Hurra, tatsächlich die Antwort ist.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:221]:
```suggestion
  "translatedText": "was, hurra, tatsächlich die Antwort ist.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 3]
```diff
@@ -227,16 +227,16 @@
...
    132.96,
    133.78
   ]
  },
  {
-  "input": "If you're wondering if that's any good, the way I heard one person phrase it is that with Wurdle four is par and three is birdie.",
+  "input": "If you're wondering if that's any good, the way I heard one person phrase it is that with wordle four is par and three is birdie.",
   "model": "nmt",
-  "translatedText": "Wenn Sie sich fragen, ob das etwas nützt: Ich habe jemanden sagen hören, dass bei Wurdle vier Par und drei Birdie sind.",
+  "translatedText": "Falls Sie sich fragen, ob das irgendetwas bringt: Ich habe gehört, wie eine Person es so ausgedrückt hat, dass bei wordle vier gleich Par und drei gleich Birdie sind.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:239]:
```suggestion
  "translatedText": "Falls Sie sich fragen, ob das irgendetwas gebracht hat: Ich habe gehört, wie eine Person es so ausgedrückt hat, dass bei wordle vier Züge einem Par und drei einem Birdie entsprechen.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 4]
```diff
@@ -281,7 +281,7 @@
  {
   "input": "And like I said, really it's an excuse for an information theory lesson.",
   "model": "nmt",
-  "translatedText": "Und wie ich schon sagte, es ist eigentlich ein Vorwand für eine Lektion in Informationstheorie.",
+  "translatedText": "Und wie ich schon sagte, es ist eigentlich ein Ausrede für eine Lektion in Informationstheorie.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:284]:
```suggestion
  "translatedText": "Und wie ich schon sagte, es ist eigentlich ein Vorwand für eine Lektion in Informationstheorie.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 5]
```diff
@@ -326,7 +326,7 @@
  {
   "input": "The thought is that if you hit a letter, you know, you get a green or a yellow, that always feels good.",
   "model": "nmt",
-  "translatedText": "Der Gedanke ist, dass wenn man einen Buchstaben trifft, man einen grünen oder einen gelben bekommt, das fühlt sich immer gut an.",
+  "translatedText": "Der Gedanke ist, dass wenn man einen Buchstaben errät, diese dann grün oder gelb markiert werden. Das fühlt sich immer gut an",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:329]:
```suggestion
  "translatedText": "Der Gedanke ist, dass wenn man einen Buchstaben errät, dieser dann grün oder gelb markiert wird. Das fühlt sich immer gut an",
```

File: 2022/wordle/german/sentence_translations.json [hunk 6]
```diff
@@ -344,7 +344,7 @@
  {
   "input": "But in these cases, even if you don't hit and you always get grays, that's still giving you a lot of information since it's pretty rare to find a word that doesn't have any of these letters.",
   "model": "nmt",
-  "translatedText": "Aber selbst wenn Sie in diesen Fällen nicht klicken und immer Grautöne erhalten, erhalten Sie dennoch viele Informationen, da es ziemlich selten ist, ein Wort zu finden, das keinen dieser Buchstaben enthält.",
+  "translatedText": "Aber selbst wenn du in diesen Fällen kein richtigen Buchstaben erräts und nur graue Buchstaben bekommst, erhälst du dennoch viele Informationen, da es ziemlich selten ist, ein Wort zu finden, das keinen dieser Buchstaben enthält.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:347]:
```suggestion
  "translatedText": "Aber selbst wenn du in diesen Fällen kein richtigen Buchstaben errätst und nur graue Buchstaben bekommst, erhältst du dennoch viele Informationen, da es ziemlich selten ist, ein Wort zu finden, das keinen dieser Buchstaben enthält.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 7]
```diff
@@ -425,7 +425,7 @@
  {
   "input": "So there's a list of words that it will allow you to enter that are considered valid guesses that's just about 13,000 words long.",
   "model": "nmt",
-  "translatedText": "Es gibt also eine Liste von Wörtern, die Sie eingeben können und die als gültige Vermutungen gelten und die nur etwa 13.000 Wörter lang sind.",
+  "translatedText": "Es gibt eine Liste von Wörtern, die Sie eingeben können und die als gültige Vermutungen gelten und die nur etwa 13.000 Wörter lang ist.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:428]:
```suggestion
  "translatedText": "Es gibt eine Liste von Wörtern, die man eingeben kann und die als gültige Vermutungen gelten. Diese ist in etwa 13.000 Wörter lang.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 8]
```diff
@@ -452,7 +452,7 @@
  {
   "input": "And in fact, there's another list of around 2300 words that are the possible answers.",
   "model": "nmt",
-  "translatedText": "Und tatsächlich gibt es noch eine weitere Liste mit rund 2300 Wörtern, die mögliche Antworten darstellen.",
+  "translatedText": "Und tatsächlich gibt es noch eine weitere Liste mit rund 2300 Wörtern, die mögliche Antworten darstellt.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:455]:
```suggestion
  "translatedText": "Und tatsächlich gibt es noch eine weitere Liste mit rund 2300 Wörtern, die mögliche Antworten darstellen.",
```

Darstellen refers to Antworten and not Liste, Antworten isi plural so we use plural form of the verb.

File: 2022/wordle/german/sentence_translations.json [hunk 9]
```diff
@@ -470,7 +470,7 @@
  {
   "input": "But what I would like to do, our challenge for this project is to see if we can write a program solving Wordle that doesn't incorporate previous knowledge about this list.",
   "model": "nmt",
-  "translatedText": "Aber was ich gerne tun würde, unsere Herausforderung für dieses Projekt besteht darin, zu sehen, ob wir ein Programm schreiben können, das Wordle löst, das keine Vorkenntnisse über diese Liste enthält.",
+  "translatedText": "Aber was ich gerne machen würde, unsere Herausforderung für dieses Projekt ist zu sehen, ob wir ein  Programm schreiben können, das Wordle löst, ohne vorheriges Wissen über diese Liste zu berücksichtigen.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:473]:
```suggestion
  "translatedText": "Aber was ich gerne als unsere Herausforderung für dieses Projekt machen würde ist zu sehen, ob wir ein Programm schreiben können, das Wordle löst, ohne dieses vorherige Wissen aus der Liste mit einzubeziehen.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 10]
```diff
@@ -515,7 +515,7 @@
  {
   "input": "So you could always just look up what tomorrow's answer will be.",
   "model": "nmt",
-  "translatedText": "Sie können also jederzeit nachschauen, wie die Antwort morgen lautet.",
+  "translatedText": "Du können also jederzeit nachschauen, wie die Antwort für morgen lautet.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:518]:
```suggestion
  "translatedText": "Du könntest also jederzeit nachschauen, wie die Antwort für morgen lautet.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 11]
```diff
@@ -560,7 +560,7 @@
  {
   "input": "Well, the reason he said he likes that unlikely W is that he likes the long shot nature of just how good it feels if you do hit that W.",
   "model": "nmt",
-  "translatedText": "Nun, der Grund, warum er sagte, dass er dieses unwahrscheinliche W mag, ist, dass ihm die Weitsicht gefällt, wie gut es sich anfühlt, wenn man dieses W trifft.",
+  "translatedText": "Nun, der Grund, warum er sagte, dass er dieses unwahrscheinliche W mag, ist die Tatsache, dass das W sehr unwahrscheinlich ist. Ist das W im Wort enthalten und man hats erraten, fühlt es sich einfach großartig an.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:563]:
```suggestion
  "translatedText": "Nun, der Grund, warum er sagte, dass er dieses unwahrscheinliche W mag, ist die Tatsache, dass das W sehr unwahrscheinlich ist. Ist das W im Wort enthalten und man hat es erraten, fühlt es sich einfach großartig an.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 12]
```diff
@@ -623,7 +623,7 @@
  {
   "input": "But at least for our first pass at all of this, let's assume that they're all equally likely and then refine that a bit later.",
   "model": "nmt",
-  "translatedText": "Aber zumindest für unseren ersten Versuch gehen wir davon aus, dass sie alle gleich wahrscheinlich sind, und verfeinern das dann etwas später.",
+  "translatedText": "Aber zumindest für unseren ersten Versuch: Lasst uns annehmen, dass sie alle gleich wahrscheinlich sind, und verfeinern das dann etwas später.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:626]:
```suggestion
  "translatedText": "Aber zumindest für unseren ersten Versuch: Lass uns annehmen, dass sie alle gleich wahrscheinlich sind, und wir verfeinern das dann etwas später.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 13]
```diff
@@ -713,7 +713,7 @@
  {
   "input": "So the most common possibility here is that you get all grays.",
   "model": "nmt",
-  "translatedText": "Die häufigste Möglichkeit besteht hier also darin, dass Sie nur Grautöne erhalten.",
+  "translatedText": "Am Wahrscheinlichsten ist es, dass du nur Grautöne erhälst.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:716]:
```suggestion
  "translatedText": "Am Wahrscheinlichsten ist es, dass du nur Graue erhältst.",
```

File: 2022/wordle/german/sentence_translations.json [hunk 14]
```diff
@@ -740,7 +740,7 @@
  {
   "input": "Or if we venture a little farther to the left, you know, maybe we go all the way over here.",
   "model": "nmt",
-  "translatedText": "Oder wenn wir uns etwas weiter nach links wagen, wissen Sie, vielleicht kommen wir bis hierher.",
+  "translatedText": "Oder wenn wir uns etwas weiter nach rechts wagen, vielleicht kommen wir dann bis hierher.",
```
Comment (zsawyer) [inline 2022/wordle/german/sentence_translations.json:743]:
```suggestion
  "translatedText": "Oder wenn wir uns etwas weiter nach links wagen, vielleicht kommen wir dann bis hierher.",
```

**Review by zsawyer — CHANGES_REQUESTED** [review]
Changes reviewed. Some changes required. Native speaker here.

**Comment by 3b1b** [comment]
Thanks for the review, for convenience I added it in https://github.com/3b1b/captions/pull/206

**Pull request closed without merging**